src/marqo/tensor_search/tensor_search.py

```diff
@@ -1577,10 +1576,10 @@ def get_query_vectors_from_jobs(
                                             f"This causes the error when we do `numpy.mean()` over all the vectors.\n"
                                             f"The original error is `{e}`.\n"
                                             f"Please check `https://docs.marqo.ai/0.0.16/API-Reference/search/#context`.")
-
             if index_info.index_settings['index_defaults']['normalize_embeddings']:
                 norm = np.linalg.norm(merged_vector, axis=-1, keepdims=True)
                 if norm > 0:
+                    # TODO: Why are we calculating norm twice?
```

Comment:
nit: this is not ideal but norm is idempotent

Reply:
but why can't we use the `norm` variable we calculated like so: `merged_vector /= norm`